tests/conftest.py

```diff
@@ -1886,6 +1913,62 @@ def format_portautoneg_test_id(param):
                     metafunc.parametrize('vlan_name', ['no_vlan'], scope='module')
 
 
+def generate_skeleton_port_info(request):
+    """
+    Return minimal port_info parameters to populate later in the format of <speed>-<category>. i.e
+
+    ["400.0-single_linecard_single_asic", "400.0-multiple_linecard_multiple_asic",...]
+    """
+    dut_info = get_snappi_testbed_metadata(request)
```

Comment:
hi @amitpawar12 , can you put this initial as 
```suggestion
    dut_info = get_snappi_testbed_metadata(request) or []
```

This is to bypass failure when running `pytest --collect-only` thank you